SUMMARY.md

```diff
@@ -35,12 +35,28 @@
 
 	*  [*Disabling and Deleting Users*](PrimeroAdministrationandConfigurationGuide.md#disabling-and-deleting-users)
 
+	*  [*Configuring Manager Dashboard*](PrimeroAdministrationandConfigurationGuide.md#configuring-manager-dashboard)
+
+	*  [*Referral Button*](PrimeroAdministrationandConfigurationGuide.md#referral-button)
+
 *  [*Configuring Programs and Modules*](PrimeroAdministrationandConfigurationGuide.md#configuring-programs-and-modules)
 
 	*  [*Configuring Programs*](PrimeroAdministrationandConfigurationGuide.md#configuring-programs)
 
 	*  [*Configuring Modules*](PrimeroAdministrationandConfigurationGuide.md#configuring-modules)
 
+*  [*System Settings Configuration*](PrimeroAdministrationandConfigurationGuide.md#system-settings-configuration)
+
+	*  [*Age Range*](PrimeroAdministrationandConfigurationGuide.md#age-range)
+
+	*  [*Reporting Location*](PrimeroAdministrationandConfigurationGuide.md#reporting-location)
+
+	*  [*Agency Code*](PrimeroAdministrationandConfigurationGuide.md#agency-code)
+
+	* [*Search for an existing ID before creating a Case record*](PrimeroAdministrationandConfigurationGuide.md#search-for-an-existing-id-before-creating-a-case-record)
```

Comment:
Need an extra space between the '*' and the '['. Only insisting because the gitbook pdf generator may not format it the same way if things are spaced differently.